starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,97 @@
+/*Some necessary adjustments*/
+
+h1, body, html, main {
+  margin: 0;
+}
+
+/*Import the font and set the default size*/
+
+html {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px
+}
+
+
+body {
+  font-family: 'Open Sans', sans-serif;
+  color: #242424;
+  font-size: 18px;
+  margin: 0;
+  background-color: #242424;
+}
+
+.main-container {
+  width: 800px;
+  margin: auto;
+  background-color: white;
+  padding-bottom: 25px;
+}
+
+.apple-header {
+  position: relative;
+  top: 0px;
+  height: 500px;
+  width: 100%;
+  background-image: url("../images/apple-pie.jpg");
```

Comment:
Background image = god idea. 
Don't think we covered this but so good use of Google.  (or experience or whatever you used)